dist/index.js

```diff
@@ -1080,9 +1080,12 @@ function objectType(value, path, resolve, traverseCallback) {
   var propertyKeys = Object.keys(properties);
   var patternPropertyKeys = Object.keys(patternProperties);
   var optionalProperties = propertyKeys.concat(patternPropertyKeys).reduce(function (_response, _key) {
-    if (requiredProperties.indexOf(_key) === -1) { _response.push(_key); }
+    if (requiredProperties.indexOf(_key) === -1) {
+       if(_key)
```

Comment:
Format this block correctly. Make sure the indentation is correct so the if statement works correctly, or put curly braces around the body.